iambic/plugins/v0_1_0/aws/utils.py

```diff
@@ -220,7 +219,7 @@ def get_current_role_arn(sts_client) -> str:
             "Unable to detect AWS Credentials. Please follow the guide here "
             "to set up AWS credentials: https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html"
         )
-        sys.exit(1)
+        raise
```

Comment:
Just saw this one - why raise here, instead of just exiting?

Reply:
I guess that the "wizard" is responsible of stop the application, and not the plugin